src/components/Messages/hooks/useMessages.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+
+import { useEffect, useState } from "react";
+import inatjs from "inaturalistjs";
+
+import { getJWTToken } from "../../LoginSignUp/AuthenticationService";
+import { MESSAGE_FIELDS } from "../../../providers/helpers";
+
+// TODO: need to trigger a rerender on change of user. Right now API parameters not being used.
+const useMessages = ( apiParams: Object ): {
+  messageList: Array<Object>,
```

Comment:
This should be `messages`, not `messageList`, since `messages` is what is being returned from the hook. Not sure if you have Flow set up, but if so you should be able to see this Flow error on line 15 of `Messages.js`.

Reply:
@albullington oh how do I get flow setup?